### ISSUE TITLE
Does update operation support nonce for the update public key

### ISSUE THREAD

**Issue by zkwzk** [decentralized-identity/ion-sdk#25]
hello ION-SDK team,

According to the https://identity.foundation/sidetree/spec/#jwk-nonce, is suggested we can implement a `nonce` that will introduce the randomness to the update commitment, so I'm wondering whether there is any implementation already support this nonce feature. Because we want to keep using the same public key to reduce the key management efforts.

**Comment by thehenrytsai** [comment]
No, it currently doesn't support it, but if the SDK does support it, don't you need to then implement the management of the `nonce` thus back to square one?

Just trying to understand and prioritize this feature ask accordingly!

**Comment by zkwzk** [comment]
hello @thehenrytsai , our use case is we will help user issue the DID, but we are expecting only maintain one restore key pair for all of the dids we issued, and for each of the user we just maintain a nonce for them, should not be very high priority
